tests/gateways/disk/test_permissions.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+import shutil
+import uuid
+import stat
+from contextlib import contextmanager
+from tempfile import mkdtemp, gettempdir
+import pytest
+from os.path import join, isfile
+from stat import S_IREAD, S_IRGRP, S_IROTH
+
+from conda.gateways.disk.update import touch
+from conda.utils import on_win
+from conda.common.compat import text_type
+from conda.gateways.disk.permissions import make_writable, recursive_make_writable
+
+
+@contextmanager
+def tempdir():
+    tempdirdir = gettempdir()
+    dirname = str(uuid.uuid4())[:8]
+    prefix = join(tempdirdir, dirname)
+    try:
+        os.makedirs(prefix)
+        yield prefix
+    finally:
+        shutil.rmtree(prefix)
```

Comment:
make this

```
finally:
    if lexist(prefix):
        rmtree(prefix)
```

Instead of `shutil.rmtree`, use `from shutil import rmtree` at the top of the file.